presto-hive/src/main/java/com/facebook/presto/hive/metastore/ThriftHiveMetastore.java

```diff
@@ -769,6 +736,41 @@ public void revokeTablePrivileges(String databaseName, String tableName, String
         }
     }
 
+    private PrivilegeBag buildPrivilegeBag(HiveMetastoreClient metastoreClient,
+                                                     String databaseName,
+                                                     String tableName,
+                                                     String grantee,
+                                                     Set<PrivilegeGrantInfo> privilegeGrantInfos)
+    {
+        PrincipalType principalType;
+        String principalName;
+
+        try {
+            if (metastoreClient.getRoleNames().contains(grantee.toLowerCase())) {
+                principalType = ROLE;
+                principalName = grantee.toLowerCase(); // Hive metastore API requires role names to be in lowercase.
```

Comment:
when using case changing methods always declare the locale as `US` otherwise you get the locale of the VM which might do strange things